### ISSUE TITLE
[notification] httr update to 1.0.0 breaks downstream package hadley/bigrquery

### ISSUE THREAD

**Issue by akhmed1** [hadley/httr#259]
Hello,

Recent update of httr to [1.0.0](https://github.com/hadley/httr/commit/485d387daffcd76aca454fad74b7b16bafd30acd) switched `httr` to `curl` instead of `RCurl`. 

This update breaks the basic functionality of `bigrquery`.

I filed the main issue with hadley/bigrquery#54 and this filing is meant to be a notification of downstream problems not captured by [cran-comments.md](https://github.com/hadley/httr/blob/master/cran-comments.md).

The issue appears to be caused by difference in behavior of curlEscape with NULLs for `curl` vs `RCurl`. 
``` r
> curl::curl_escape(NULL)
Error in curl::curl_escape(NULL) :
    `url` must be a character vector.
> RCurl::curlEscape(NULL)
character(0)
```

It appears `bigrquery` relies on the default `RCurl` behavior and does not have tests that would show that the package is now broken downstream.

**Comment by craigcitro** [comment]
fixed by jeroenooms/curl#32.

**Comment by craigcitro** [comment]
@hadley i think this one's good to close
